cmake/configs/default.cmake

```diff
@@ -387,8 +387,8 @@ hunter_default_version(thrift VERSION 0.11.0-p0)
 hunter_default_version(tinydir VERSION 1.2-p0)
 hunter_default_version(tinyxml2 VERSION 6.2.0-p1)
 hunter_default_version(toluapp VERSION 1.0.93-p1)
-hunter_default_version(tomcrypt VERSION 1.17-p3)
-hunter_default_version(tommath VERSION 1.0.1-p0)
+hunter_default_version(tomcrypt VERSION 1.18.2-p0)
+hunter_default_version(tommath VERSION 1.0-p2)
```

Comment:
Why?

Reply:
Why are  macOS builds broken?

Reply:
> hunter_default_version(tommath VERSION 1.0-p2)

Please don't touch `tommath`, this pull request is about `tomcrypt`.

Reply:
ok

Reply:
(tommath VERSION 1.0-p2) ok

Reply:
> (tommath VERSION 1.0-p2) ok

Can you please clarify what you mean?

Reply:
i've  changed it by accident at merge

Reply:
We see eye to eye